DabraHalleloEndaAbunaSamuel/DHAS015.xml

```diff
@@ -0,0 +1,128 @@
+                                       </support>
+                                       <extent>                                          
+                                                       <measure unit="leaf"/>                                  
+                                                       <dimensions type="outer" unit="mm">
+                                                           <height cert="low">300</height>
+                                                           <width cert="low">240</width>
+                                                           <depth cert="low">60</depth>
+                                                           
+                                           </dimensions>
+                                       </extent>   
+                            
+                            <condition key="good"/>
+                            
+                         </supportDesc>
+            
+                         <layoutDesc>
+                             <layout columns="2" cert="high">                                 
+                             </layout>
+                         </layoutDesc>
+                         
+                     </objectDesc>
+                       <handDesc>
+                           <handNote xml:id="h1" script="Ethiopic">
+                               <seg type="script">Recent.</seg>
+                               <seg type="ink">Black; red.</seg>                                
+                           </handNote>                           
+                       </handDesc>                       
+                       
+                       
+                         <bindingDesc>
+                             <binding contemporary="false">
+                                 <decoNote xml:id="b1" type="Boards" cert="medium">Wooden boards.</decoNote>
+                                 <decoNote xml:id="b2" type="Cover" cert="medium">Blind-tooled leather cover, 
+                                     <term key="textileInlay">textile inlays</term>.</decoNote>
+                                 <decoNote xml:id="b3" type="SewingStations" cert="medium">Two pairs of sewing stations.</decoNote>
+                                 <decoNote xml:id="b4" type="bindingMaterial" cert="medium">
+                                     <material key="wood"/>
+                                     <material key="leather"/>
+                                     <material key="textile"/>
+                                 </decoNote>
+                                 
+                             </binding>
+                         </bindingDesc>
+                   </physDesc>                     
+                  
+                    
+                      <history>
+                         <origin>
+                           <origDate notBefore="1950" notAfter="2010">Recent time</origDate>.                              
```

Comment:
I think, it is more precise to say: "Second half of 20th century or beginning of 21st century, but not after 2010". What is recent today will change over time.

Reply:
I changed, but avoiding mentioning exact years.